src/App.js

```diff
@@ -1,50 +1,115 @@
-import './App.css';
-import Counter from './components/Counter';
-import SearchForm from './components/SearchForm';
-import GenreSelect from './components/GenreSelect';
+import MovieTile from './components/MovieTile';
+import MovieDetails from './components/MovieDetails';
+import SortControl from './components/SortControl';
+import { useState } from 'react';
 
 function App() {
-  const genres = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
-  const selectedGenre = 'All';
+  const [sortOrder, setSortOrder] = useState('releaseDate');
```

Comment:
It is better to keep such things (I mean string "releaseDate") in separate place and do not hardcode it in the component. Data structure could look like this but is up to you
`const sortOrders = [
 {
 value: 'releaseDate',
 formatterValue: 'Release date'
},
 {
 value: 'releaseDate',
 formatterValue: 'Release date'
},
]`

Reply:
So just don't use a string "releaseDate", but use an array and have useState use that array?

Reply:
as example
![image](https://user-images.githubusercontent.com/19514890/230194075-1c211fb5-3472-4197-bbe6-4b62586a2788.png)